kubectx.plugin.zsh

```diff
@@ -4,6 +4,7 @@
 # oh-my-zsh compatible plugins.
 
 KUBECTX_DIR="$(dirname $0)/kubectx"
+OH_MY_ZSH_DIR="$(dirname $0)/../../.."
```

Comment:
This assumes that people are only using oh-my-zsh and its directory structure. On my zgen install for example, `OH_MY_ZSH_DIR` would end up being `/Users`, which my user doesn't have write access to.

Reply:
I am quite new to **zsh** in general so I kinda  missed this... Since this variable is only used in completion I am removing it, check comment below